src/util.ts

```diff
@@ -3,3 +3,6 @@ export const nopSource = {
 }
 
 export const nopSink = async (_: any) => {}
+
+const charset = Array.from('ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/')
```

Comment:
Is there a specific reason to include `+` and `/`? I'd keep it simple and only use alphanumeric characters or perhaps only hex ones. Not having `/` makes it e.g. easier to split off the random bytes of the full ufrag, you can be sure it only contains a single `/` from the `libp2p+webrtc+v1/` prefix.

Reply:
This is a requirement in the spec: https://github.com/libp2p/specs/blob/master/webrtc/README.md?plain=1#L95

Reply:
Yes, the prefix is a requirement. But I don't see a requirement that the random characters must contain `+`  and `/`.

Reply:
It's not a hard requirement but the choice of characters is from the default set of characters used to generate a ufrag. I don't see a reason to change that. As far as splitting off the random bytes is concerned, we are guaranteed that there is at least one `/` and that all bytes after it are random.

Reply:
Thanks, I wondered where this choice came from, but it's from https://www.rfc-editor.org/rfc/rfc8839#section-5.4, where it states it can be `ice-char` which is exactly that list (https://www.rfc-editor.org/rfc/rfc8839#section-5.1). That makes sense then.